py-polars/src/series/buffers.rs

```diff
@@ -281,3 +277,135 @@ fn get_boolean_buffer_length_in_bytes(length: usize, offset: usize) -> usize {
         n_bytes + 1
     }
 }
+
+#[pymethods]
+impl PySeries {
+    /// Construct a PySeries from information about its underlying buffers.
+    #[staticmethod]
+    unsafe fn _from_buffers(
+        dtype: Wrap<DataType>,
+        data: Vec<PySeries>,
+        validity: Option<PySeries>,
+    ) -> PyResult<Self> {
+        let dtype = dtype.0;
+        let data = data.to_series();
+
+        match data.len() {
+            0 => {
+                return Err(PyTypeError::new_err(
+                    "`data` input to `from_buffers` must contain at least one buffer",
+                ));
+            },
+            1 if validity.is_none() => {
+                let data = data.into_iter().next().unwrap();
```

Comment:
nit: `.pop().unwrap()` or `.first().unwrap()` is a bit more directly accessing

Reply:
Changed to `.pop().unwrap()`!